### PR TITLE
Add user coreydaley as an approver

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -17,6 +17,7 @@ aliases:
   - LalatenduMohanty # Over the Air Updates
   - Miciah # Network Edge
   - adambkaplan # Build + Jenkins
+  - coreydaley # Build + Jenkins
```
Comment (bparees) [inline OWNERS_ALIASES:20]: remove adam while you're in here

File: test/extended/OWNERS [hunk 2]
```diff
@@ -16,3 +17,4 @@ approvers:
   - ironcladlou
   - adambkaplan
   - danwinship
+  - coreydaley
```
Comment (bparees) [inline test/extended/OWNERS:20]: you shouldn't need this level of approval, add yourself to the test/extended/$subdirs your team owns tests or library code in.

**Review by bparees — COMMENTED** [review]
you should remove adam in places where you are replacing his role

**Comment by coreydaley** [comment]
@bparees updated, ptal

**Comment by coreydaley** [comment]
@bparees Can you skip this failing test?

**Comment by bparees** [comment]
> @bparees Can you skip this failing test?

https://docs.ci.openshift.org/docs/how-tos/overriding-failing-ci-jobs/

```
Confirm that someone/team owns and is aware of the failure you are trying to override. Don’t just override something broken and let the rest of the org keep running into it until someone else reports it. You don’t have to solve it, but at least make sure there’s an active/current bug or that the team in question acknowledges the issue.
```

**Comment by coreydaley** [comment]
@bparees 
Opened bug https://bugzilla.redhat.com/show_bug.cgi?id=2087285 for this test that is continuously failing.

**Pull request merged**